owltorrent-core/src/main/java/edu/rice/owltorrent/core/serialization/TorrentParser.java

```diff
@@ -26,14 +27,16 @@
   public static final String filesField = "files";
   public static final String lengthField = "length";
   public static final String pathField = "path";
+  public static final Bencoder bencoder = new Bencoder();
 
+  private static byte[] infoPart;
   /**
    * Bencode a Torrent file and extract important attributes
    *
    * @param file Torrent file path
    * @return pruned Torrent object
    */
-  public static Torrent parse(@NonNull File file) throws IOException {
+  public static Torrent parse(@NonNull File file) throws Exception {
```

Comment:
why was this changed to generic Exception? (Note that we don't need to specify throwing RuntimeExceptions like BTException. )